specification/advisor/resource-manager/readme.md

```diff
@@ -26,7 +26,31 @@ These are the global settings for the Advisor API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2025-05-preview
+tag: package-2025-08-preview
+```
+
+### Tag: package-2025-08-preview
+These settings apply only when `--tag=package-2025-08-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2025-08-preview'
+input-file:
+  - Microsoft.Advisor/preview/2025-08-01-preview/advisor.json
+  - Microsoft.Advisor/preview/2025-08-01-preview/assessments.json
+  - Microsoft.Advisor/preview/2025-08-01-preview/resiliencyReviews.json
+suppressions:
+  - code: ArmResourcePropertiesBag
+    reason: Suppressing for now to avoid breaking change.
+    from:
+      - assessments.json
+    where:
+      - $.definitions["AssessmentResult"]
+  - code: RequiredPropertiesMissingInResourceModel
```

Comment:
>RequiredPropertiesMissingInResourceModel [](http://example.com/codeflow?start=10&length=40)

This is critical RPC violation even for preview version. Please provide justification for suppression (why are required property missing in the first place, besides continuation of existing violation to avoid breaking change), long term plan,technical reason why it can't be addressed in this preview version?